bitnami/minio/templates/provisioning-configmap.yaml

```diff
@@ -54,4 +54,20 @@ data:
     }
   {{- end }}
   {{- end }}
+  {{- range $policy := .Values.provisioning.policies }}
+  policy-{{ $policy.name }}.json: |
+      {{- $statementsLength := sub (len $policy.statements) 1 }}
+      {
+        "Version": "2012-10-17",
+        "Statement": [
+          {{- range $i, $statement := $policy.statements }}
+          {
+            "Effect": "{{ default "Allow" $statement.effect }}"{{ if $statement.actions }},
```

Comment:
for safety concerns wouldn't be better to have `Deny` as default?

Reply:
👍 Agreed. This would also align with the default value in [AWS IAM](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_elements_effect.html).